hikari/impl/rest.py

```diff
@@ -3160,7 +3160,7 @@ async def fetch_application_commands(
         else:
             route = routes.GET_APPLICATION_GUILD_COMMANDS.compile(application=application, guild=guild)
 
-        response = await self._request(route)
+        response = await self._request(route, query={"with_localizations": "true"})
```

Comment:
Note for me: Check when and if they are going to remove this and make it the default